rdoc/en/grammar.en.rdoc

```diff
@@ -4,25 +4,24 @@
 
 == Class Block and User Code Block
 
-There's two block on toplevel.
-one is 'class' block, another is 'user code' block. 'user code' block MUST
-places after 'class' block.
+There are two top-level blocks: the 'class' block, and the 'user code'
+block. The 'user code' block MUST be after the 'class' block.
 
 == Comment
 
-You can insert comment about all places. Two style comment can be used,
-Ruby style (#.....) and C style (/*......*/) .
+You can insert comment about all places. Two comment styles can be used,
```

Comment:
"You can insert comment about all places" could also be improved.

Everything else looks good.

Reply:
Done.